### DISCUSSION TITLE
Is there a way to enable performance summary using environment variables or any other way?

### DISCUSSION THREAD

**Discussion by pubudu91** [dotnet/msbuild#10426] [Q&A]
I know we can enable performance summary by passing `clp:PerformanceSummary` when calling MSBuild through the CLI. But is there a way to enable it other than passing it as a CLI arg? e.g., throug environment variables.

**Comment by rainersigwald** [comment]
No, I don't think so. Can you elaborate on why you want that? There may be something else we could suggest.

**Reply by pubudu91** [reply]
So I'm working with a wrapper around MSBuild which I have no control over. This wrapper doesn't support providing arbitrary arguments to it. So there's no way for me to pass the `/clp:PerformanceSummary` arg to MSBuild. I was wondering if there's an environment variable that I could set which would achieve the same thing (enabling performance summary). I've been using the bin log with the log viewer and observing the run times for targets through that. But being able to get just the performance summary as a text would've been great.
